.travis.yml

```diff
@@ -19,7 +20,7 @@ before_install:
     - conda info -a
 
 install:
-    - conda create -q -n test_env python=$TRAVIS_PYTHON_VERSION numpy scipy pandas matplotlib nose
+    - conda create -q -n test_env python=$TRAVIS_PYTHON_VERSION numpy scipy==0.15 pandas matplotlib nose
```

Comment:
I think you're right about that - however, it doesn't change the end result... to be continued!